bentoml/_internal/log.py

```diff
@@ -38,17 +47,13 @@ class TraceFormatter(Formatter):
     """
 
     def __init__(self):
-        Formatter.__init__(
-            self,
-            fmt=TRACED_LOG_FORMAT,
-            datefmt=DATE_FORMAT,
-        )
-        self.control_formmater = Formatter(SERVICE_LOG_FORMAT, datefmt=DATE_FORMAT)
+        Formatter.__init__(self, fmt=TRACED_LOG_FORMAT, datefmt=DATE_FORMAT)
```

Comment:
Should this be super.__init?

Reply:
yep

Reply:
![Screenshot 2022-05-28 at 02 03 42](https://user-images.githubusercontent.com/29749331/170812509-f9bc7e80-2c2f-4002-b2a6-5f5b15bba5cb.png)

Reply:
![Screenshot 2022-05-28 at 02 04 21](https://user-images.githubusercontent.com/29749331/170812524-e3ab141c-f8b9-450d-b332-fc11d1d43435.png)

Reply:
That's because we have `self` to be the first argument.
Just "super().__init__(fmt=xxx, datefmt=xxx)" works, I think.